go-server/go.mod

```diff
@@ -8,5 +8,6 @@ require (
 	github.com/gocarina/gocsv v0.0.0-20201208093247-67c824bc04d4
 	github.com/gorilla/mux v1.8.0
 	github.com/joho/godotenv v1.3.0
+	github.com/segmentio/kafka-go v0.4.8
```

Comment:
I installed kafka about commenting out when it is called in the code, let me know if it gives you trouble

Reply:
This is working fine for me 👍